libs/coin-modules/coin-solana/src/deviceTransactionConfig.ts

```diff
@@ -139,6 +145,54 @@ function fieldsForCreateATA(command: TokenCreateATACommand): DeviceTransactionFi
 
   return fields;
 }
+
+function fieldsForCreateApprove(command: TokenCreateApproveCommand): DeviceTransactionField[] {
+  const fields: Array<DeviceTransactionField> = [];
+
+  fields.push({
+    type: "address",
+    label: "Approve token acct",
```

Comment:
Should we check with product on what to show for both approve and revoke ?
```suggestion
    label: "Approve token account",
```